applications/FluidDynamicsApplication/custom_conditions/monolithic_wall_condition.h

```diff
@@ -258,6 +258,8 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MonolithicWallCondition : public Co
             rRightHandSideVector.resize(LocalSize);
 
         noalias(rRightHandSideVector) = ZeroVector(LocalSize);
+        MatrixType LHS;
+        this->ApplyNeumannCondition(LHS, rRightHandSideVector);
```

Comment:
To reduce code duplication, I would suggest creating the temporary LHS and calling the corresponding LHS+RHS method (CalculateLocalSystem I think?).

Also: I am assuming that here we are just adding the constant (External fixed pressure type) part of the boundary condition... If I remember correctly, terms that depend on the current velocity should go into the "DampingMatrix" contribution function.